src/main/java/com/uwetrottmann/tmdb2/Tmdb.java

```diff
@@ -17,14 +51,16 @@
 import retrofit2.Retrofit;
 import retrofit2.converter.gson.GsonConverterFactory;
 
+import java.io.IOException;
+
 /**
  * Helper class for easy usage of the TMDB v3 API using retrofit.
- *
+ * <p>
```

Comment:
That is kind of the point 😁 The empty line inside the source file is just for clarity, but should not show up in the actual javadoc.

The setting to change this is:
![image](https://cloud.githubusercontent.com/assets/888674/26436593/ed55a01c-4117-11e7-8e72-dc7ce35e04b3.png)